scripts/machine-launch-script.sh

```diff
@@ -12,6 +12,10 @@ sudo yum install -y python36 patch diffstat texi2html texinfo subversion chrpath
 sudo yum install -y gtk3-devel
 # deps for firesim-software (note that rsync is installed but too old)
 sudo yum install -y python36-pip python36-devel rsync
+# Install GNU make 4.x (needed to compile riscv-glibc 2.28+)
```

Comment:
We need to announce this very publicly in the appropriate channels.

Reply:
Yeah.  On further thought, not everyone might be using the AMI, so it is probably better to enable devtoolset-7 only if it exists and the make version is too old.